src/compiler/crystal/macros.cr

```diff
@@ -755,7 +755,7 @@ module Crystal::Macros
     def map : ArrayLiteral
     end
 
-    # Similar to `Hash#[]`
+    # Similar to `Hash#[]?`
```

Comment:
I wanted to change it to `#[]?` but I guess that would be a breaking change. Perhaps the documentation should say '# Equal to `Hash#[]?` now?

Reply:
The breaking change isn't a big issue. We could add the new method and deprecate the old.

But I'm not sure that's a good idea... renaming to `#[]?` is just more complicated. Unless there's a need for both, a raising and non-raising variant, I think it should keep it as it is.